youtube_dl/extractor/ndtv.py

```diff
@@ -9,7 +13,7 @@
 
 
 class NDTVIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?ndtv\.com/video/(?:[^/]+/)+[^/?^&]+-(?P<id>\d+)'
+    _VALID_URL = r'https?://(?:(?:www|gadgets|khabar|profit)\.)?ndtv\.com/videos?/(?:[^/]+/)*[^/?^&]+-(?P<id>\d+)'
```

Comment:
There are more formats:
https://sports.ndtv.com/cricket/videos/2nd-t20i-rock-thrown-at-australia-cricket-team-bus-after-win-over-india-469764
http://food.ndtv.com/video-basil-seeds-coconut-porridge-419083?pfrom=home-videos
https://auto.ndtv.com/videos/the-cnb-daily-october-13-2017-469935

Reply:
First two need slightly different regex.  Third doesn't seem to have `__filename` string.

Reply:
While we are at it.  Adding three more:
https://doctor.ndtv.com/videos/why-a-green-and-cracker-free-diwali-is-healthy-for-you-469936
http://movies.ndtv.com/videos/cracker-free-diwali-wishes-from-karan-johar-kriti-sanon-other-stars-470304
https://swirlster.ndtv.com/video/how-to-make-friends-at-work-469324

Reply:
Instead of trying to enumerate `(?:[^/]+\.)?ndtv\.com` is ok.